dashboard/app/models/user.rb

```diff
@@ -497,12 +497,16 @@ def self.from_omniauth(auth, params)
 
       # clever provides us these fields
       if user.user_type == TYPE_TEACHER
-        # if clever told us that the user is a teacher, we just trust
-        # that they are adults; we don't actually care about age
```

Comment:
This is being removed because our not caring about age has nothing to do with Clever, see `:make_teachers_21` (run before save).